### ISSUE TITLE
`separable areas` link is broken

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/wai-wcag-em#90]
In WCAG-EM 1.0, it links to [Website with Separable Areas](https://www.w3.org/TR/WCAG-EM/#separable)

In this updated version, it also links to `#separable` but the anchor does not exist - https://w3c.github.io/wai-wcag-em/#separable

**Comment by hidde** [comment]
Thanks, we removed that type of website as it is much more commonplace these days that websites have separable areas. The link should be gone now in the latest version.
